  This file activates the doc stuff
  Do in the top directory

     patch -p0 < doc/activate.patch

  To revert it

     patch -Rp0 < doc/activate.patch

  Don't worry about these lines, patch can handle them

Index: Makefile.am
===================================================================
RCS file: /cvs/conge/Makefile.am,v
retrieving revision 1.18
diff -u -r1.18 Makefile.am
--- Makefile.am	25 Jun 2003 20:00:28 -0000	1.18
+++ Makefile.am	2 Jul 2003 20:04:39 -0000
@@ -1,4 +1,4 @@
-SUBDIRS = src po data examples pixmaps glade
+SUBDIRS = src po data doc examples pixmaps glade
 
 EXTRA_DIST = BUGS	\
   .diffexclude		\
Index: configure.in
===================================================================
RCS file: /cvs/conge/configure.in,v
retrieving revision 1.30
diff -u -r1.30 configure.in
--- configure.in	24 Jun 2003 16:44:02 -0000	1.30
+++ configure.in	2 Jul 2003 19:54:12 -0000
@@ -127,6 +127,8 @@
 AC_OUTPUT(Makefile 
           po/Makefile.in
 	  data/Makefile
+	  doc/Makefile
+	  doc/C/Makefile
 	  glade/Makefile
 	  pixmaps/Makefile
 	  examples/Makefile 
Index: src/cong-menus.c
===================================================================
RCS file: /cvs/conge/src/cong-menus.c,v
retrieving revision 1.15
diff -u -r1.15 cong-menus.c
--- src/cong-menus.c	6 Jun 2003 12:45:29 -0000	1.15
+++ src/cong-menus.c	6 Jun 2003 14:37:26 -0000
@@ -1154,6 +1154,20 @@
 
 }
 
+static void menu_callback_help(gpointer callback_data,
+				guint callback_action,
+				GtkWidget *widget)
+{
+	GError *error = NULL;
+
+	gnome_help_display("conglomerate.xml", NULL, &error);
+
+	if(error!=NULL)
+	{
+		g_warning(error->message);
+		g_error_free(error);
+	}
+}
 
 /* The menus, for a window that contains a document: */
 #define TOOLS_MENU_ACTION_MARKER (1)
@@ -1228,9 +1242,7 @@
 	{ N_("/_Tools"),        NULL, NULL, TOOLS_MENU_ACTION_MARKER, "<Branch>" },
 
 	{ N_("/_Help"),        NULL, NULL, 0, "<Branch>" },
-#if ENABLE_UNIMPLEMENTED_MENUS
-	{ N_("/Help/_Contents"), "F1", unimplemented_menu_item, 0, "<StockItem>",GTK_STOCK_HELP },
-#endif /* #if ENABLE_UNIMPLEMENTED_MENUS */
+	{ N_("/Help/_Contents"), "F1", menu_callback_help, 0, "<StockItem>",GTK_STOCK_HELP },
 	{ N_("/Help/_About"),    NULL, menu_callback_about, 0, "<StockItem>", GNOME_STOCK_ABOUT }
 
 };
@@ -1257,9 +1269,7 @@
 #endif /* #if ENABLE_DEBUG_MENU */
 
 	{ N_("/_Help"),        NULL, NULL, 0, "<Branch>" },
-#if ENABLE_UNIMPLEMENTED_MENUS
-	{ N_("/Help/_Contents"), "F1", unimplemented_menu_item, 0, "<StockItem>",GTK_STOCK_HELP },
-#endif /* #if ENABLE_UNIMPLEMENTED_MENUS */
+	{ N_("/Help/_Contents"), "F1", menu_callback_help, 0, "<StockItem>",GTK_STOCK_HELP },
 	{ N_("/Help/_About"),    NULL, menu_callback_about, 0, "<StockItem>", GNOME_STOCK_ABOUT }
 
 };
